src/components/groupSet/GroupSetSelect.js

```diff
@@ -21,37 +21,54 @@ const GroupSetSelect = ({
     value,
     allowNone,
     onChange,
-    errorText,
     className,
 }) => {
     const { nameProperty } = useCachedDataQuery()
     const { loading, error, data } = useDataQuery(ORG_UNIT_GROUP_SETS_QUERY, {
         variables: { nameProperty },
     })
 
+    const ITEM_NONE = useMemo(() => ({ id: 'none', name: i18n.t('None') }), [])
```

Comment:
Curious why useMemo is needed here. With no dependencies, this would never change woud it?